src/lib/Organism/Footer.svelte

```diff
@@ -0,0 +1,157 @@
+<!-- Footer -->
+<footer class="footer">
+    <div class="container-footer">
+        <div class="footer-content">
+            <div class="footer-logo">
+            </div>
+            <div class="footer-links">
+                <ul class="footer-menu">
+                    <li><a href="https://hallostrandeiland.nl/contact">Contact</a></li>
+                    <li><a href="https://hallostrandeiland.nl/privacy-local">Privacy</a></li>
+                    <li><a href="https://gebiedonline.nl">Ontwikkeld met software van coöperatie Gebiedonline</a></li>
+                </ul>
+            </div>
+            <div class="footer-social">
```

Comment:
Hier is teveel genest in div-element. Graag refactoren!

Reply:
Toegepast

```svelte
<!-- Footer -->
<footer class="footer">
    <div class="container-footer">
        <section class="footer-content">
            <nav class="footer-links">
                <ul class="footer-menu">
                    <li><a href="https://hallostrandeiland.nl/contact">Contact</a></li>
                    <li><a href="https://hallostrandeiland.nl/privacy-local">Privacy</a></li>
                    <li><a href="https://gebiedonline.nl">Ontwikkeld met software van coöperatie Gebiedonline</a></li>
                </ul>
            </nav>
            <section class="footer-social">
                <ul class="social-icons">
                    <!-- Facebook -->
                    <li><a href="/">[Facebook SVG]</a></li>
                    <!-- Twitter -->
                    <li><a href="https://twitter.com/?lang=nl">[Twitter SVG]</a></li>
                    <!-- Instagram -->
                    <li><a href="https://www.instagram.com/">[Instagram SVG]</a></li>
                    <!-- LinkedIn -->
                    <li><a href="https://nl.linkedin.com/">[LinkedIn SVG]</a></li>
                </ul>
            </section>
        </section>
        <div class="footer-bottom">
            <p>&copy; 2024 Hallo Strandeilanden.</p>
        </div>
    </div>
</footer>
```
 Ik heb <nav> gebruikt voor de navigatie links, wat helpt bij zoekmachineoptimalisatie en toegankelijkheid. Het <section> element voor de sociale media links groepeert deze op een betekenisvolle manier.